TheMET/TheMET/Base.lproj/Main.storyboard

```diff
@@ -99,6 +99,21 @@
             </objects>
             <point key="canvasLocation" x="708" y="632"/>
         </scene>
+        <!--About App View Controller-->
```

Comment:
You do all the layout inside the `AppViewController` itself. So, I guess you don't need this controller in a storyboard.